pkg/controller/kieapp/constants/constants.go

```diff
@@ -260,6 +260,16 @@ const (
 	//Envs
 	KubeNS     = "KUBERNETES_NAMESPACE"
 	KubeLabels = "KUBERNETES_LABELS"
+
+	//Kafka extension
+	KieServerKafkaExtEnabled          = "KIE_SERVER_KAFKA_EXT_ENABLED"
```

Comment:
besides tests where else is this constants being used?

Reply:
Just for test, moving in the tests